eos/form-factors/parametric-bgjvd2019.cc

```diff
@@ -39,16 +39,16 @@ namespace eos
         _mBar(p[prefix + "::mBar@HQET"], *this),
         _a(p[prefix + "::a@HQET"], *this),
         _opt_lp_model(o, "model-lp", { "power-series", "exponential" }, "power-series"),
-        _opt_lp_zorder(o, "z-order-lp", { "2", "3", "4", "5" }, "3"),
-        _enable_lp_z3(1.0 ? _opt_lp_zorder.value() >= "3" : 0.0),
-        _enable_lp_z4(1.0 ? _opt_lp_zorder.value() >= "4" : 0.0),
-        _enable_lp_z5(1.0 ? _opt_lp_zorder.value() >= "5" : 0.0),
-        _opt_slp_zorder(o, "z-order-slp", { "1", "2" }, "2"),
-        _enable_slp_z2(1.0 ? _opt_slp_zorder.value() >= "2" : 0.0),
-        _opt_sslp_zorder(o, "z-order-sslp", { "0", "1", "2" }, "1"),
-        _enable_sslp_z1(1.0 ? _opt_sslp_zorder.value() >= "1" : 0.0),
-        _enable_sslp_z2(1.0 ? _opt_sslp_zorder.value() >= "2" : 0.0),
-        _opt_sslp_limit(o, "SU3F-limit-sslp", { "0", "1" }, "0"),
```

Comment:
@mreboud Careful, this change to using ``BooleanOption`` modified the behaviour of existing analyses.
The reason is that ``destringify<bool>`` interprets only ``true`` as a true boolean value, and anything else is interpreted as ``false``, including ``1``. But the latter was used in lieu of ``true`` here. We need to simultaneously adapt the (private) analysis 2019-04 and any ongoing analysis.

@gubernari Could you please have a look at this as soon as possible please and adapt your ongoing analysis file?

Reply:
@dvandyk thanks for spotting this; I thought `1.0` would be interpreted as `true`.
@gubernari Understood, although this should be completely safe.

Reply:
Concerning your question, I would say no. Let's change the analysis file instead and make sure we use BooleanOption in the future.